Greenshot/Drawing/Surface.cs

```diff
@@ -1371,6 +1404,25 @@ public Image GetImageForExport()
 			return GetImage(RenderMode.EXPORT);
 		}
 
+		private Rectangle ZoomClipRectangle(Rectangle rc, double scale)

```

Comment:
You can make this, and the next method, static as they don't use any of the instance variables.

Reply:
I'm so used to VS suggestions on this I didn't pay much attention myself. And got caught when it didn't catch these methods for some reason